USER_MANUAL.md

```diff
@@ -42,16 +42,36 @@ Actors used in gyms are in `Contetnt\Actors`: add any new Actors to this directo
 ##### Empty gym
 * The template for creating new gyms. Copy this to use as a starting point for your own gym.
 
+##### Dynamic Components gym
+* Demonstrates that:
+  * Dynamic component are correctly added to and removed from a replicated Actor (with load-balancing enabled).
+* Contains:
+  * A character with a PlayerController with key bindings for adding and removing a component.
+* To setup the gym:
+  * Adjust the settings in "SpatialOS Settings -> Editor Settings -> Launch -> Launch configuration file options -> Server Workers" to include 4 servers in a 2x2 grid.
+    * Set both "Rectangle grid column count" and "Rectangle grid row count" to 2
+    * Set "Instances to launch in editor" to 4
+* To test the gym:
+    * Move the character until the inspector shows that the EntityACL and other server simulated components are on different workers
```

Comment:
Is there no way to set this up so that it is forced? May be beyond the scope of this PR (or should be an engine net test)

Reply:
Not afaik depends on the runtime load-balancing the EntityACL, not sure it's worth looking into fixing really, since it takes 10 seconds to run between workers and find the right setup